torch/csrc/jit/passes/liveness.h

```diff
@@ -0,0 +1,22 @@
+#pragma once
+
+#include <ATen/ATen.h>
+#include <ATen/core/ivalue.h>
+#include <ATen/core/jit_type.h>
+#include <ATen/core/stack.h>
+#include <torch/csrc/WindowsTorchApiMacro.h>
+#include <torch/csrc/jit/ir.h>
+#include <unordered_map>
+#include <list>
+#include <vector>
+#include <c10/util/sparse_bitset.h>
+namespace torch {
+namespace jit {
+
+using ::c10::ProfiledTensorTypePtr;
+using SparseBitVector = ::c10::SparseBitVector<256>;
+
```

Comment:
Comment descibing the form of the output? Is that live in or live out of a node?